pontoon/gpt/openai_connection.py

```diff
@@ -0,0 +1,51 @@
+from openai import OpenAI
+
+
+class OpenAIService:
+    def __init__(self):
+        self.client = OpenAI()
+
+    def get_translation(
+        self, english_text, translated_text, characteristic, target_locale_name
```

Comment:
We should raise a `ValueErorr` if `target_locale_name` (should be `target_language_name`) doesn't exist in the `pontoon.base.models.Locale` table.